test/support/rummager_helpers.rb

```diff
@@ -89,21 +89,22 @@ def stub_organisations_for_taxon(content_id, organisations)
     )
   end
 
-  def generate_search_results(count)
+  def generate_search_results(count, supergroup="default")
     (1..count).map do |number|
-      rummager_document_for_slug("content-item-#{number}")
```

Comment:
Instead of having a method for each supergroup, can we not continue to use the `rummager_document_for_slug` method, passing in the format we want? E.g:

```
rummager_document_for_slug("content-item-#{number}", format = "local_transaction")
```

Maybe I'm missing something?